member_database/events/__init__.py

```diff
@@ -326,6 +365,7 @@ def participants(event_id):
 
 
 @events.route('/<int:event_id>/write_mail/', methods=['GET', 'POST'])
+@route_name_to_id('/<string:name>/write_mail/', methods=['GET', 'POST'])
```

Comment:
More magic... but would it be possible to have the decorate not need the arguments? E.g. like this:

```
@route_name_to_id
@events..route('/<int:event_id>/write_mail/', methods=['GET', 'POST'])
```

would somehow get the route and the options and do something like `route.replace("int:event_id", "string:name")`?

Reply:
Wondered the same thing! 
Should be possible... flask registers the function names so there is definitely a method like get get_route_rule(function) :D

Reply:
@maxnoe events.url_map.iter_rules() should do the trick...